Makefile

```diff
@@ -53,7 +53,7 @@ release-local:
 
 .PHONY: test-e2e
 test-e2e:
-	$(GO) test -timeout 1h -count 1 -v -tags e2e -run "$(TEST)" ./test/e2e
+	$(GO) test -timeout 2h -count 1 -v -tags e2e -run "$(TEST)" ./test/e2e
```

Comment:
This is the only TBD change. When https://github.com/openshift/cluster-ingress-operator/pull/756 gets introduced, than this can go away (so regardless, there will be 1 more change)